### PR TITLE
feat: standard language section in YIP 0000 

### PR DESCRIPTION
**Pull request by nksazonov**

### CODE REVIEW

**Comment by mnaichuk** [comment]
Great, thank you!

**Review by mnaichuk — APPROVED** [review]

File: docs/YIP/YIP-0000-yips.md [hunk 1]
```diff
@@ -12,45 +12,59 @@ The Yellow Improvement Proposal (YIP) system is being introduced as an essential
...
+_Shall_, _should_, _may_ and _can_ are defined in [6.4.7](https://standards.ieee.org/about/policies/opman/sect6.html#6.4.7) of the _IEEE SA Standards Board Operations Manual_.
 
-## Status
+The word _shall_ indicates mandatory requirements strictly to be followed in order to conform to the standard and from which no deviation is permitted (_shall_ equals _is required to_).
 
-What is the status, such as proposed, accepted, rejected, deprecated, superseded, etc.?
+Note that the word _must_ is deprecated and shall not be used when stating mandatory requirements; _must_ is used only to describe unavoidable situations. The word _will_ is deprecated and shall not be used when stating mandatory requirements; _will_ is only used in statements of fact.
 
-## Context
+The word _should_ indicates that among several possibilities, one is recommended as particularly suitablewithout mentioning or excluding others; or that a certain course of action is preferred but not necessarilyrequired (_should_ equals _is recommended that_).
```
Comment (dpatsora) [inline docs/YIP/YIP-0000-yips.md:56]: "suitable without"

File: docs/YIP/YIP-0000-yips.md [hunk 2]
```diff
@@ -12,45 +12,59 @@ The Yellow Improvement Proposal (YIP) system is being introduced as an essential
...
+> This is the template in [Documenting architecture decisions - Michael Nygard](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).
+> You can use [adr-tools](https://github.com/npryce/adr-tools) for managing the YIP files.
+>
+> In each YIP file, write these sections:
+>
+> ### Title
+>
+> #### Status
+>
+> What is the status, such as proposed, accepted, rejected, deprecated, superseded, etc.?
```
Comment (dpatsora) [inline docs/YIP/YIP-0000-yips.md:32]: Can we add "in progress"?

File: docs/YIP/YIP-0000-yips.md (hunk 1 above)
Comment (dpatsora) [reply docs/YIP/YIP-0000-yips.md:56]: "necessarily required"

**Review by dpatsora — CHANGES_REQUESTED** [review]

File: docs/YIP/YIP-0000-yips.md (hunk 1 above)
Comment (nksazonov) [reply docs/YIP/YIP-0000-yips.md:56]: Quiet a catchy eye you have 😉

**Review by dpatsora — APPROVED** [review]

**Pull request merged**